src/methods/validate-fixture-input.ts

```diff
@@ -49,14 +82,33 @@ export function validateFixtureInput(
           const responseKey = node.alias?.value || node.name.value;
 
           const fieldDefinition = typeInfo.getFieldDef();
-          const fieldType = fieldDefinition?.type;
+          if (fieldDefinition === undefined || fieldDefinition === null) {
+            errors.push(`Cannot validate ${responseKey}: missing field definition`);
+            return BREAK;
+          }
+          const fieldType = fieldDefinition.type;
 
           for (const currentValue of currentValues) {
             const valueForResponseKey = currentValue[responseKey];
 
             // Field is missing from fixture
             if (valueForResponseKey === undefined) {
-              errors.push(`Missing expected fixture data for ${responseKey}`);
+              const parentType = typeInfo.getParentType();
+              if (!parentType) {
+                // This shouldn't happen with a valid query and schema - TypeInfo should always
+                // provide parent type information when traversing fields. This check is here to
+                // satisfy TypeScript's type requirements (getParentType() can return null).
+                errors.push(`Cannot validate ${responseKey}: missing parent type information`);
```

Comment:
Do we need this? I'm guessing we were using `parentType` before but now we're not?